scripts/leaderboard.coffee

```diff
@@ -118,6 +156,19 @@ module.exports = (robot) ->
     field[name] = score
     return "#{name}'s score set to #{score}."
 
+  getRank = (score) ->
+    for i in [0..15]
+      if score <= pointThresholds[i]
+        return ranks[i]
+
+    return ranks[15]
+
+  isLeveledUp = (score) ->
+    for i in [0..15]
+      if score == pointThresholds[i] + 1
```

Comment:
Will this work if there are multiple plusplus in a message that lead to level up and `score - pointThresholds >= 2`?

Reply:
Multiple ++ are handled individually and a message is posted for every ++ so this does work in that case. I tested this on my test workspace.

Reply:
Just tested. Works as intended